tests/data/test_entryexitanalysis.py

```diff
@@ -0,0 +1,94 @@
+        'indicator_list': "bb_upperband,ema_10"
+    })
+    patch_exchange(mocker)
+    result1 = pd.DataFrame({'pair': ['ETH/BTC', 'LTC/BTC'],
+                            'profit_ratio': [0.0, 0.0],
+                            'profit_abs': [0.0, 0.0],
+                            'open_date': pd.to_datetime(['2018-01-29 18:40:00',
+                                                         '2018-01-30 03:30:00', ], utc=True
+                                                        ),
+                            'close_date': pd.to_datetime(['2018-01-29 20:45:00',
+                                                          '2018-01-30 05:35:00', ], utc=True),
+                            'trade_duration': [235, 40],
+                            'is_open': [False, False],
+                            'stake_amount': [0.01, 0.01],
+                            'open_rate': [0.104445, 0.10302485],
+                            'close_rate': [0.104969, 0.103541],
+                            "is_short": [False, False],
+                            'enter_tag': ["enter_tag_long", "enter_tag_long"],
+                            'exit_reason': [ExitType.ROI, ExitType.ROI]
+                            })
+
+    backtestmock = MagicMock(side_effect=[
+        {
+            'results': result1,
+            'config': default_conf,
+            'locks': [],
+            'rejected_signals': 20,
+            'timedout_entry_orders': 0,
+            'timedout_exit_orders': 0,
+            'canceled_trade_entries': 0,
+            'canceled_entry_orders': 0,
+            'replaced_entry_orders': 0,
+            'final_balance': 1000,
+        }
+    ])
+    mocker.patch('freqtrade.plugins.pairlistmanager.PairListManager.whitelist',
+                 PropertyMock(return_value=['ETH/BTC', 'LTC/BTC', 'DASH/BTC']))
+    mocker.patch('freqtrade.optimize.backtesting.Backtesting.backtest', backtestmock)
+
+    patched_configuration_load_config_file(mocker, default_conf)
+
+    args = [
+        'backtesting',
+        '--config', 'config.json',
+        '--datadir', str(testdatadir),
+        '--strategy-path', str(Path(__file__).parents[1] / 'strategy/strats'),
+        '--timeframe', '5m',
+        '--timerange', '1515560100-1517287800',
+        '--export', 'signals',
+        '--cache', 'none',
+        '--strategy-list',
+        'StrategyTestV3Analysis',
+    ]
+    args = get_args(args)
+    start_backtesting(args)
+
+    captured = capsys.readouterr()
+    assert 'BACKTESTING REPORT' in captured.out
+    assert 'EXIT REASON STATS' in captured.out
+    assert 'LEFT OPEN TRADES REPORT' in captured.out
+
+    args = [
+        'analysis',
+        '--config', 'config.json',
+        '--datadir', str(testdatadir),
+        '--analysis_groups', '0',
+        '--strategy',
+        'StrategyTestV3Analysis',
+    ]
+    args = get_args(args)
+    start_analysis_entries_exits(args)
+
+    captured = capsys.readouterr()
+    assert 'enter_tag_long' in captured.out
```

Comment:
That last CI failure was most likely caused by this new test (even though the failure was in a different place).

Please either use the `backtesting_cleanup` fixture (imported from test_backtesting) - or call `Backtesting.cleanup()` manually after the test to ensure backtesting is cleaned up properly.